core/gather/driver/target-manager.js

```diff
@@ -153,6 +171,38 @@ class TargetManager extends ProtocolEventEmitter {
     }
   }
 
+  /**
+   * @param {LH.Crdp.Runtime.ExecutionContextCreatedEvent} event
+   */
+  _onExecutionContextCreated(event) {
+    // This execution context was made via the protocol (e.g. by us or Puppeteer).
+    if (event.context.origin === '://' || event.context.origin === '') return;
+    // Just in case the above changes somehow.
+    if (event.context.name === '__puppeteer_utility_world__') return;
+    if (event.context.name === 'lighthouse_isolated_context') return;
```

Comment:
Can we drop the general case filtering above if the narrow cases by name here work? For instance, an extension uses the protocol to create an execution context, we'd probably still want to capture unload listeners.